packages/graph/hash_graph/tests/test_data/entity/book.json

```diff
@@ -1,5 +1,5 @@
 {
-  "https://blockprotocol.org/@alice/types/property-type/name": "The Time Machine",
+  "https://blockprotocol.org/@alice/types/property-type/name/": "The Time Machine",
```

Comment:
Having the `BaseURI` like this looks weird to me, we should allow both ways.

Reply:
We discussed this offline and we need some further changes in the upstream crate.
Summary: we need (and require) the trailing slash.